collector/logical_disk.go

```diff
@@ -171,26 +170,25 @@ func (c *LogicalDiskCollector) Collect(ctx *ScrapeContext, ch chan<- prometheus.
 // - https://msdn.microsoft.com/en-us/library/ms803973.aspx - LogicalDisk object reference
 type Win32_PerfRawData_PerfDisk_LogicalDisk struct {
```

Comment:
Bit of a nitpick, but now that we don't use WMI here, we don't need to have the type name match the WMI class name. We also don't need to export the type, so perhaps change to just `logicalDisk`?

Reply:
Yes, better to clean it up now then to leave others wondering at the type name in the future.
Are you happy to leave the documentation link for the type?

Reply:
I think the doc link still looked relevant, but I didn't look super closely, is there something off with it?

Reply:
Not at all. My preference would be to leave it in, as it is still largely relevant for the perflib metrics. Just checking.

Reply:
Type has been renamed to `logicalDisk`. I've rebased the commits and force pushed them to my branch.